dev/tasks/tasks.yml

```diff
@@ -1486,15 +1486,14 @@ tasks:
       image: debian-go
   {% endfor %}
 
-  # be sure to update binary-task.rb when upgrading ubuntu
+  # be sure to update binary-task.rb when upgrading Debian
   test-debian-12-docs:
     ci: github
     template: docs/github.linux.yml
     params:
       env:
         JDK: 17
       pr_number: Unset
-      flags: "-v $PWD/build/:/build/"
```

Comment:
What happens if the user doesn't pass the flag? Does it simply build the docs inside the container?

Reply:
Yes.